02_classes-objects-methods/02_03_car.py

```diff
@@ -6,3 +6,27 @@
 #
 # Create at least two different objects of this `Car()` class and demonstrate
 # changing the objects' attributes.
+
+class Car:
+
+    def __init__(self, model, year, max_speed) -> None:
+        self.model = model
+        self.year = year
+        self.max_speed = max_speed
+
+    def newspeed(self):
+        self.max_speed += 5
+        return self.max_speed
+    
+    def detail(self):
+        return f'This car model is {self.model} made in {self.year} and its max speed is {self.max_speed} miles per hour.'
```

Comment:
it would make more sense to just have this be the `__str__` function, because it's purpose is to allow you to print out details.